src/Test/Desktop/OpenRiaServices.Common.Test/SandBoxer.cs

```diff
@@ -19,7 +19,7 @@ namespace OpenRiaServices.Common.Test
     /// </summary>
     public class SandBoxer : MarshalByRefObject
     {
-
+#if NETFRAMEWORK
```

Comment:
Why?

Reply:
The code only works on NETFRAMEWORK since it uses medium trust etc.

But I think we should be able to delete the file since we don't really support medium trust anymore